### DISCUSSION TITLE
"Add single comment" Dismisses the Team Review

### DISCUSSION THREAD

**Discussion by jacobangel** [community/community#37994] [Pull Requests]
What's happening: 
A teammate adds my github team as a reviewer.
I add a comment with the "Add single comment" button 
The PR says I have reviewed on behalf of the team, it removes the team reviewer, and puts me as a reviewer.

What I expected
That the team tag would only disappear if I started an actual review

Why would I do this?
I am not an IC and reading the code and just asking questions, not necc. reviewing on behalf of the team. 

Is this expected? Is there any way to get to my expected workflow?

**Comment by ossanna16** [comment]
Hi there @jacobangel and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by pyrtsa** [comment]
Duplicate of #5289. (Annoying indeed!)

**Reply by romoore** [reply]
I don't think this is strictly a duplicate of #5289. That one wants to keep a Team as a reviewer even after someone on the team adds a Review. This one is pointing out that a Single Comment is not a Review and shouldn't be treated as such.

I often provide "drive-by" comments as a team member, but know others have more contextual information and should be the "real" Reviewers. My "Add Single Comment" should _not_ be marking me as a "Reviewer".
